geth-dev-node/Dockerfile

```diff
@@ -1,64 +1,19 @@
 FROM ethereum/client-go:latest
-COPY genesis.json /app/genesis.json
-RUN echo "p" > /app/password.txt
-RUN geth makedag 0 /app/.ethash
+
+RUN mkdir /home/\.ethereum
+RUN mkdir /home/\.ethereum/keystore
+RUN mkdir /home/\.ethash
```

Comment:
Consider `/geth` instead of `/home` to match `/parity` in the other node and to also make it more clear to the reader what is in the folder.  Also consider not using `.` prefixed folder names since those don't show up in `ls`.

Reply:
Sounds good.  I'll make this change.